PeriodogramAnalysis/PeriodogramAnalysis.py

```diff
@@ -319,23 +305,21 @@ def computing_welch_bandpower(self, signal, band, channel, window_sec=None, rela
         band : list
             Frequency band of interest.
         channel : int
-            Channel number to compute bandpower for.
-        window_sec : float
-            Window length in seconds for Welch's method.
+            Channel number to compute band power for.
         relative : bool
-            If True, compute relative bandpower.
+            If True, compute relative band power.
 
         Returns:
         --------
         float
-            Computed bandpower.
+            Computed band power.
         """
         low, high = band
         local_signal = signal.data
         frequency = signal.rate
 
-        if window_sec is not None:
-            nperseg = window_sec * frequency
+        if self.window_length_for_welch is not None:
+            nperseg = self.window_length_for_welch * frequency 
         else:
             nperseg = (2 / low) * frequency
         freqs, psd = sig.welch(local_signal[:, channel], frequency, nperseg=nperseg)
```

Comment:
Just use `signal[channel]`. No need to redefine `local_signal`

Reply:
Sure, have modified it.

Reply:
I don't see you changed it properly. Can you just use `signal[channel]` instead of `signal.data[:,channel]`?

Reply:
Have changed. I thought it won't work previously...